sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v1/init.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.search_terms_derived.suggest_impression_sanitized_v1`
+PARTITION BY
+  DATE(submission_timestamp)
+CLUSTER BY
+  normalized_channel,
+  sample_id
+OPTIONS
+  (require_partition_filter = TRUE, partition_expiration_days = 15)
```

Comment:
At some point I think I'd like the expiration days in `metadata.yaml` as well, perhaps for some future state where terraform deploys these tables from metadata and they are incidentally populated by query. I see `require_partition_filter` and clustering info are already there.